repo/packages/D/drone-server/0/marathon.json.mustache

```diff
@@ -0,0 +1,124 @@
+{
+  "acceptedResourceRoles": null,
+  "args": null,
+  "backoffFactor": 1.15,
+  "backoffSeconds": 1,
+  "cmd": null,
+  "constraints": [],
+  "container": {
+    "docker": {
+      "forcePullImage": false,
+      "image": "drone/drone:0.5",
+      "network": "BRIDGE",
+      "parameters": [],
+      "portMappings": [
+        {
+          "containerPort": 8000,
+          "hostPort": 0,
+          "labels": {},
+          "protocol": "tcp",
+          "servicePort": 0
+        }
+      ],
+      "privileged": false
+    },
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
```

Comment:
according to [this](http://readme.drone.io/admin/installation-guide/) the `drone-server` doesn't need the docker socket